Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -167,6 +167,8 @@ def is_option_set(self):
     def is_nested(self):
         return 'Nested' in self.attributes
 
+    def is_webkit_platform(self):
```

Comment:
Having EnumTraits and serialization.in will cause linker issues for MSVC.  For now let's just remove all this file's changes, and switch NotificationPayload to only having EnumTraits until we can figure out what to do in a separate PR.

Reply:
Would I expect to see that breakage on an EWS? Currently looks like it's fine?

Reply:
EWS passed, but I'll make that change anyways

Reply:
These changes to this file are no longer used or tested.